example/README.MD

```diff
@@ -14,4 +14,8 @@ To get this project running on iOS just follow these steps
 
 1. `yarn` or `npm i`
 2. `yarn start` or `npm start`
-3 `react-native run-ios`
+3. `react-native run-ios`
```

Comment:
In markdown, to do sequential numbers you just keep using '1.', like so:

1. first thing
1. second thing
1. third thing

markdown magic